cypress/support/fragments/inventory/inventorySearchAndFilter.js

```diff
@@ -502,10 +502,6 @@ export default {
     cy.expect(inventorySearchAndFilterInput.has({ value: option }));
   },
 
-  verifySelectedSearchOption(option) {
```

Comment:
there are tests that use this method

Reply:
yes, but take a look at rows 501 - 503. It is the same exact method. Don't know how it got there, but a good thing to delete a duplicate.